styles/all/template/event/overall_header_stylesheets_after.html

```diff
@@ -9,6 +9,9 @@
 	{% if S_REGISTERED_USER -%}
 	ga('set', 'userId', {{ GOOGLEANALYTICS_USER_ID }});
 	{% endif -%}
+	{% if S_ANONYMIZE_IP -%}
```

Comment:
to delete the extra white spaces that are added after the twig statements, so the rendered JS code is all nicely aligned in the HTML source.

Reply:
It keeps code from looking like:
```
ga('create', 'UA-0000000-00', 'auto');
	ga('set', 'userId', 2);
		ga('send', 'pageview');
```

Reply:
Oh I see. Good hack! BTW any chance this also gets geo check and is only set for EEU IPs?

Reply:
I doubg that's needed. besides, this isn't so much about anonymizing users, as it is about giving the board owner the option to do this. It's board owners from EU that need to comply with this. Board owners outside EU don't need this, no matter where their users come from. So with this, EU operated forums can enable this feature and feel better about complying with their country's laws 😄

Reply:
Alright. But you've got one thing wrong: even if board owner is not from EU, if he has a user from EU, he is required to either get a consent or anonymize IP.

Reply:
Well, we don't have to worry about consent within this extension, anomyzing IP is enough. Using Google's method of anonymizing all IPs is fine, no need to try to hack it for EU only since that probably wouldn't be fool proof.